src/routers/report.js

```diff
@@ -0,0 +1,21 @@
+const {Router} = require('express');
+const router = Router();
+const checkAuth = require('../middleware/auth');
+const checkRoleAuth = require('../middleware/role-auth');
+const {getReports, postReport, getReport, getReportsFilters} = require('../controllers/report-controller');
+const {validationReport, validationReportFilters, validationCriterion, validationReportId}= require('../validators/report-validator');
+const {validateExistsAccounts} = require('../controllers/account-controller');
+
+router.get('/reports/:filter/:criterion', checkAuth, checkRoleAuth(["manager"]), validationReportFilters, validationCriterion, getReportsFilters);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
router.get("/reports/:filter/:criterion", checkAuth, checkRoleAuth(["manager"]), validationReportFilters, validationCriterion, getReportsFilters);
```